src/f32/vec2.rs

```diff
@@ -199,6 +199,14 @@ impl Vec2 {
         self.length().recip()
     }
 
+    /// Computes the Euclidean distance between two points.
+    #[inline]
+    pub fn distance(self, other: Vec2) -> f32 { (self - other).length() }
```

Comment:
Why are you putting the function bodies all on one line?

Reply:
Just used cargo fmt to format all the codes.

Thanks for the heads up!